app/experimenter/nimbus-ui/README.md

```diff
@@ -236,15 +236,29 @@ While an experiment is in `draft` status all fields are technically allowed to b
 
 No additional action is required; as long as you correctly set the form group's `controlId` and the form error's `name` these review-readiness messages will automatically appear as needed when a user attempts to go into review with incomplete fields.
 
-## Access Visualization Data Locally
+## Results Page and Visualization Data
 
 The Results page renders experiment analysis data and can only be accessed when an experiment is complete, the analysis feature flag is on, and the analysis data returned from the visualization API endpoint is available.
 
+### Locally Access Visualization Data for Experiments in Prod
+
+At the time of writing, locally created complete experiments don't return data from the visualization endpoint. However, it is possible to view a locally created experiment that mirrors an experiment in production to test the Results page locally with production visualization data.
+
 To see the Results page and data locally:
 
 1. Ensure your [Google Credentials](https://github.com/mozilla/experimenter#google-credentials) are configured
 2. Set `FEATURE_ANALYSIS=true` in your local `.env` file
-3. Create a Nimbus Experiment at `localhost/admin` with experiment slug "testing-nimbus" status "complete," a branch/branch slug named "control", and another branch/branch slug named "treatment". Alternately, change an existing, complete Nimbus experiment with branches in place to reflect the experiment slug `testing-nimbus`.
+3. Find a completed experiment in production that has analysis data ready that you'd like to test your changes against
+4. At `localhost/admin`, add a new Nimbus Experiment with the same experiment slug as the production experiment you'd like to test against. Change the status to "complete," and add every branch (name and slug) to match the production experiment. Also, make sure "reference branch" is set to the reference branch for that production experiment.
+5. Optionally change `fetch_jetstream_data` in `app/experimenter/settings.py` to `30` so it fetches and stores the data in the local database
```

Comment:
I think this is not optional (unless you do a force refresh like you describe below). The data won't show up for 8 hours if you don't lower the time it takes to fetch

Reply:
Oh I see these instructions are for fetching from GCP.  You can also test locally by just pulling the `results_data` from the prod admin and dumping it into the local admin.  Depends on what you're trying to test.  If you're testing the frontend you don't need to worry about the jetstream task, you can just skip this step and dump the processed data right into the db.  If you want to test the jetstream task you gotta do this.  Probably worth describing both?

Reply:
If someone doesn't have access to prod admin, they can probably also just copy the entire response from the endpoint on the experiment they want to look at. I actually haven't ever done it that way by just dumping it via local admin, I've just been following the instructions here so that's definitely seems worth noting, I'll add👍

Reply:
@jaredlockhart turns out if you do it that way, the slug and branches don't even need to match. 😆 PR has been updated.